home/views.py

```diff
@@ -10,16 +11,21 @@ def landingpage(request):
     return render(request, 'home/landingpage.html')
 
 
-def displayQuestion(request, **kwargs):
+def displayQuestionPage(request, **kwargs):
+    form = CommentForm(request.POST)
     question = Question.objects.all().get(id=kwargs['pk'])
     sortAnsBy = request.GET["sortanswersby"] if 'sortanswersby' in request.GET else ''
     context = {
         "question": question,
         "answers": question.get_answers_feed(sortAnsBy),
         "answersCount": question.answer_set.count(),
         "tags": question.tags.values(),
-        "title": question.get_question_title()
+        "title": question.get_question_title(),
+        "form": form
     }
+    if request.method == 'POST':
+        profile = Profile.objects.filter(user=request.user).first()
+        return addAnswer(form, profile, question)
```

Comment:
Great job,
After PR #65  will be merged you can replace these lines with return `addAnswer(form, request.profile, question)`